### PR TITLE
adding delete orderItem

### PR DESCRIPTION
**Pull request by tounbukto**

### CODE REVIEW

**Review by marinedemonchauxuca — APPROVED** [review]

File: backend_nestjs/dining-service/src/table-orders/controllers/table-orders.controller.ts [hunk 1]
```diff
@@ -59,9 +59,16 @@ export class TableOrdersController {
   @ApiUnprocessableEntityResponse({ type: TableOrderAlreadyBilledException, description: 'TableOrder is already billed' })
   @Post(':tableOrderId')
   async addMenuItemToTableOrder(@Param() getTableOrderParams: GetTableOrderParams, @Body() addMenuItemDto: AddMenuItemDto): Promise<TableOrder> {
+    console.log('passing by here iam a controller');
```
Comment (GabrielCogne) [inline backend_nestjs/dining-service/src/table-orders/controllers/table-orders.controller.ts]: Can you remove or change this log for something a bit more useful? 😓

File: backend_nestjs/dining-service/src/table-orders/services/table-orders.service.ts [hunk 2]
```diff
@@ -57,7 +58,7 @@ export class TableOrdersService {
 
   async addOrderingLineToTableOrder(tableOrderId: string, addMenuItemDto: AddMenuItemDto): Promise<TableOrder> {
     const tableOrder: TableOrder = await this.findOne(tableOrderId);
-
+    console.log(tableOrder);
```
Comment (GabrielCogne) [inline backend_nestjs/dining-service/src/table-orders/services/table-orders.service.ts]: Can you add something to the log to say it is about adding item

Comment (tounbukto) [reply backend_nestjs/dining-service/src/table-orders/services/table-orders.service.ts]: that was for debugging xD

File: backend_nestjs/dining-service/src/table-orders/controllers/table-orders.controller.ts (hunk 1 above)
Comment (tounbukto) [reply backend_nestjs/dining-service/src/table-orders/controllers/table-orders.controller.ts]: debugging

**Pull request merged**